instrumentation/openai/openai-java-1.1/library/src/main/java/io/opentelemetry/instrumentation/openai/v1_1/GenAiAttributes.java

```diff
@@ -15,6 +15,7 @@ final class GenAiAttributes {
 
   static final class GenAiOperationNameIncubatingValues {
     static final String CHAT = "chat";
+    static final String EMBEDDING = "embeddings";
```

Comment:
maybe a nitpick, but should this be `EMBEDDINGS` to match the value? Maybe not, since it's singular in all the other class names